suprkewl-bot/ext/util.py

```diff
@@ -278,7 +282,9 @@ async def serverbanner(self, ctx):
             await ctx.send("This guild has no banner!")
         else:
             await ctx.send(ctx.guild.banner_url_as(format="png"))
-
+    @commands.command()
+    async def hastebin(self, ctx, text):
+        await ctx.send(bot.post_to_hastebin(text))
```

Comment:
You're missing whitespace above and below this.
The command signature should be `(self, ctx, *, text)`.
Also, look through other parts of the codebase and you'll see that there's error handling involved when using this function.
`post_to_hastebin` is a coroutine.
Finally, there was no need for the import madness because `ctx.bot` exists.